### PR TITLE
shalinee | TEST | worked on responsive on TB,PL,BS

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by aditi-tiwari17 — APPROVED** [review]

File: apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.ts [hunk 1]
```diff
@@ -63,6 +63,8 @@ export class TbPlBsFilterComponent implements OnInit, OnDestroy {
     public currentBranch: any = { name: '', uniqueName: '' };
     /** Stores the current company */
     public activeCompany: any;
+    /** */
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.ts:66]: JS doc missing

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.ts (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.ts]: Done

File: apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.ts [hunk 2]
```diff
@@ -63,7 +63,7 @@ export class TbPlBsFilterComponent implements OnInit, OnDestroy {
     public currentBranch: any = { name: '', uniqueName: '' };
     /** Stores the current company */
     public activeCompany: any;
-    /** */
+    /** stores tthe mobile screen value */
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.ts:66]: Incorrect spelling and format `True, if mobile screen size is detected`

File: apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.ts [hunk 3]
```diff
@@ -63,7 +63,7 @@ export class TbPlBsFilterComponent implements OnInit, OnDestroy {
...
     public isMobileScreen: boolean = true;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.ts:67]: @shalinee-walkover why is it by default true?

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**